### PR TITLE
refactor(RHTAPREL-626): remove ReleaseStrategy from all tests

### PR DESCRIPTION
**Pull request by johnbieren**
A recent API refactoring removed the ReleaseStrategy CRD and moved its information into other CRDs. This commit removes the ReleaseStrategy CRs and references from all tests in this repository.

### CODE REVIEW

**Comment by johnbieren** [comment]
This is blocked by the API changes merging in infra-deployments. It is also untested for the same reason

File: rh-push-to-registry-redhat-io/release-resources/release-plan-admission.yaml [hunk 1]
```diff
@@ -6,6 +6,27 @@ metadata:
...
+  policy: rh-push-to-registry-redhat-io-test-policy
+  serviceAccount: release-service-account
+  data:
+    pyxis:
+      server: stage
+      secret: pyxis
+    images:
+      addGitShaTag: true
+    mapping:
+      components:
```
Comment (johnbieren) [inline rh-push-to-registry-redhat-io/release-resources/release-plan-admission.yaml:30]: Awaiting reply from @happybhati on this one. The mapping in the ReleaseStrategy doesn't exist, so not sure how it worked or what the intended mapping is

File: fbc-stage-index/release-resources/release-plan.yaml [hunk 2]
```diff
@@ -8,5 +8,4 @@ metadata:
   namespace: dev-release-team-tenant
 spec:
   application: e2e-fbc-stage-index-application
-  displayName: e2e-fbc-stage-index-releaseplan
```
[the remark this answers is not in the document]
Comment (johnbieren) [reply fbc-stage-index/release-resources/release-plan.yaml:11]: displayName is no longer part of the ReleasePlan spec https://github.com/redhat-appstudio/release-service/commit/9de00d233f1f4b3c925c95170ca0031a41324ccb#diff-7fc47651f81f69e7be2b0ad09ac6d3e6e85c7f1edfe1ca3989abaaf103b123afL27

[the remark this answers is not in the document]
Comment (mmalina) [reply fbc-stage-index/release-resources/release-plan.yaml:11]: OK, thanks.

**Review by davidmogar — APPROVED** [review]

**Review by scoheb — APPROVED** [review]

**Review by happybhati — APPROVED** [review]

**Review by scoheb — APPROVED** [review]

File: rh-push-to-registry-redhat-io/release-resources/release-plan-admission.yaml (hunk 1 above)
Comment (scoheb) [reply rh-push-to-registry-redhat-io/release-resources/release-plan-admission.yaml:30]: if it works as-is, then we are good.

**Pull request merged**